### ISSUE TITLE
disjoint paths example

### ISSUE THREAD

**Issue by ojnas** [Telecominfraproject/oopt-gnpy#159]
Running:
`python path_requests_run.py meshTopologyExampleV2.json meshTopologyExampleV2_services.json`

results in the following output:
```
CRITICAL:gnpy.core.request:No disjoint path found with added constraint
No disjoint path found with added constraint
Computation stopped.
```
This is not surprising since `meshTopologyExampleV2.json` seems to specify a tree topology without possible disjoint paths. Maybe in this case it's better to not request disjoint paths in `meshTopologyExampleV2_services.json` and instead introduce another example that illustrates the disjoint paths feature? (In meshTopologyToy.xls in the examples folder there is currently no  requests for disjoint paths.)

**Comment by EstherLerouzic** [comment]
I have proposed a change to address this issue

**Comment by dutc** [comment]
Fixed in #161
